cfn-resources/trigger/test/cfn-test-create-inputs.sh

```diff
@@ -18,22 +32,20 @@ if [[ "$*" == help ]]; then usage; fi
 rm -rf inputs
 mkdir inputs
 
-project_Id="${2:-625454459c4e6108393d650d}"
-db_name="${3:-store}"
-coll_name="${4:-sales}"
+project_Id="${2:-63f4df9e1c744217893c19f7}"
+db_name="${3:-sample_mflix}"
+coll_name="${4:-comments}"
 trigger_name="salestrigger-${RANDOM}"
-func_name="${5:-cfn_func}"
-func_id="${6:-63862553ac0702272aa701ba}"
-service_id="${7:-6387aee08659af5254b0a51e}"
-app_id="${7:-638624a5167f5659feb75971}"
+func_name="${5:-myFuncInTrigger}"
+func_id="${6:-64020cfc5ef00ef0be9584e4}"
+service_id="${7:-64020111d61a66df7ea2c732}"
+app_id="${7:-64020111d61a66df7ea2c72b}"
```

Comment:
[q] Shouldn't these IDs being input params?

Reply:
Because there are so many parameters I think it's it's okay to have default values if these are not passed as input params. Another approach would be to have env variables for each of these. Open to suggestions.

Reply:
Since most of these params are IDs and must be changed for each run (I cannot use these values if I run the test since I don't have access to your Atlas Org ), we should make them input params.

Reply:
added input params defaulting to env variables (for publishing)

Reply:
Could you also create a Jira ticket to apply this change to other scripts? We should ensure we can publish resources without manually changing the test scripts. Thanks!